test/user_test.rb

```diff
@@ -64,4 +64,32 @@ def test_user_has_organizers
       assert_kind_of Chiketto::Organizer, @user.organizers.first
     end
   end
+
+  def test_user_has_contact_lists
+    find_me
+
+    VCR.use_cassette 'user-contact-lists' do
+      assert_kind_of Chiketto::ContactList, @user.contact_lists.first
+    end
+  end
+
+  def test_user_find_contact_list_returns_contact_list
+    find_me
+
+    VCR.use_cassette 'user-find-contact-list' do
+      assert_kind_of Chiketto::ContactList, @user.find_contact_list(411987)
```

Comment:
As I see it, the biggest issue with VCR here is that it records the api key
in the request, which is probably not something you want to share publicly.
Even creating a sandbox account at eventbrite just for testing against has
its risks (unless they provide a public one or something). It might be
worth converting the existing VCR cassettes to plain old webmock stubs, but
with a fake api key and ids substituted. Those stubs can be run in test
setup so you never have to worry about the api key's validity, eventbrite
having api availability issues, internet connection being down, etc... You
do run the risk of a developer creating a bad mock at some point and ending
up with passing tests on code that doesn't work, but the api response
structure is documented well enough that that should be easily managed.

For real safety, you could take things a step further and create an
additional path for running tests against the live api instead of the
mocked responses. Just a different test setup that, instead of setting up
the webmock stubs, configures a valid api key, user id, contact id, etc...
That would let you occasionally run the whole test suite against the live
api, providing a nice validation of the mock data and unit tests that
double as integration tests.